engine/battle_anims/functions.asm

```diff
@@ -3995,7 +4112,9 @@ Functionce672:
 	ld [hl], a
 	ret
 
-BattleAnimFunction_4B:
+BattleAnimFunction_Unused1:
```

Comment:
We're [transitioning](https://github.com/pret/pokecrystal/wiki/Code-cleanup#unusedunreferencedstubs) from `Unused` in symbols to `; unused` as comments. How about `BattleAnimFunction_AnimObjB0`?